src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -427,6 +434,50 @@ func (c controller) validateConsolePod() bool {
 	return false
 }
 
+func (c controller) waitForClusterVersion() error {
+	func1 := func() bool {
```

Comment:
`func1`? maybe `getClusterVersionAndUpdateProgress`?

Reply:
np. will change that.

Reply:
Done